### ISSUE TITLE
DNS prefetching?

### ISSUE THREAD

**Issue by Fruglemonkey** [alex/what-happens-when#181]
In some browsers [such as chromium](https://www.chromium.org/developers/design-documents/dns-prefetching#TOC-Omnibox) perform a DNS prefetch before you hit enter. The moment 'google.com' gets suggested/autocompleted, a DNS request will already have been sent off.

Possible addition?

**Comment by Amar-Chaudhari** [comment]
I dont think DNS prefetching works that way. DNS prefetching requires html tags to tell it to prefetch DNS records. 
 For example - `<link rel="dns-prefetch" href="//example.com">`

**Comment by strugee** [comment]
@Amar-Chaudhari no, look at the link in the original description. Omnibox queries are prefetched in Chromium.

**Comment by Amar-Chaudhari** [comment]
@strugee   
Yup, you are correct !  
I guess it's also better to know how pre-fetching works inside a html page as that is what most other browsers are going to use.
